frontend/src/components/authentication/CreateAccountForm.jsx

```diff
@@ -0,0 +1,310 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import validateName, { validateEmail, validatePassword } from "../../helpers/regexValidator";
+import "../../css/create-account-form.css";
+
+const styles = {
+  createAccountContent: {
+    padding: "12px 32px 0 32px"
+  },
+  inputTitle: {
+    padding: "12px 0 6px 0",
+    fontWeight: "bold"
+  },
+  inputs: {
+    width: "100%",
+    padding: "3px 6px 3px 6px",
+    borderRadius: 4
+  },
+  zoneForNames: {
+    display: "flow-root"
+  },
+  inputForNames: {
+    width: 190,
+    padding: "3px 6px 3px 6px",
+    borderRadius: 4
+  },
+  iconForNames: {
+    color: "#278400",
+    position: "absolute",
+    margin: "8px 0 0 170px"
+  },
+  iconForOtherFields: {
+    color: "#278400",
+    position: "absolute",
+    margin: "8px 0 0 370px"
+  },
+  loginBtn: {
+    width: 150,
+    display: "block",
+    margin: "24px auto"
+  },
+  validationError: {
+    color: "red",
+    marginTop: 6
+  }
+};
+
+let SUBMIT_BTN_DISABLED;
+
+class LoginForm extends Component {
```

Comment:
The class names you are using do not match the name of the file.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/c69b9a2f0170c5941645d863a72f8391adf869cb